nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/FlowController.java

```diff
@@ -3279,14 +3256,14 @@ public synchronized void onLeaderElection() {
         });
     }
 
+
     /**
      * Sets whether this instance is clustered. Clustered means that a node is either connected or trying to connect to the cluster.
      *
      * @param clustered true if clustered
      * @param clusterInstanceId if clustered is true, indicates the InstanceID of the Cluster Manager
-     * @param clusterManagerDn the DN of the NCM
      */
-    public void setClustered(final boolean clustered, final String clusterInstanceId, final String clusterManagerDn) {
+    public void setClustered(final boolean clustered, final String clusterInstanceId) {
```

Comment:
Given that this is non-public API and is only used by StandardFlowService do you mind if we make it package private? Don't need to actually do it as I can do it during merge/squash. Just want your OK.

Basically since we alredy started conversation around annotation private APIs, annotation is the last resort for cases where native Java compile constructs can't protect. And I see this as a case. We can always go back to public if there is justification in the future.